### PR TITLE
Add connection node info to response when creating a child node

### LINKED ISSUE #70: Upon "child node" creation, return enough information to connect to the node
Currently, when you create a child node, the response object is:

https://github.com/L2-Technology/sensei/blob/main/proto/sensei.proto#L115

`pubkey` and `macaroon`.  Both of which are awesome things to have.

However, to actually connect to the node, we need the rest of the connection string.  Currently this is only available in the `ListNode` object (https://github.com/L2-Technology/sensei/blob/main/proto/sensei.proto#L40), which is retrievable by paging through the `ListNodes` request/responses.

It would be wonderful if we could surface the full node info both in the `CreateNodeResponse` and the `Info`/`GetInfo` for a given node.

https://github.com/L2-Technology/sensei/blob/main/proto/sensei.proto#L331

### PR DESCRIPTION
**Pull request by orbitalturtle**
Adds connection info in proto response when creating a child node. Closes #70

### CODE REVIEW

File: proto/sensei.proto [hunk 1]
```diff
@@ -115,6 +115,8 @@ message CreateNodeRequest {
 message CreateNodeResponse {
     string pubkey = 1;
     string macaroon = 2;
+    string listen_addr = 8;
+    int32 listen_port = 9;
```
Comment (johncantrell97) [inline proto/sensei.proto:119]: can we also add the node id. 

also i'm not actually familiar with this numbering or how it works but I think maybe it needs to/ should go in order?

**Comment by johncantrell97** [comment]
thanks! 

also need to make sure you update the `http` api to include this information as well

File: proto/sensei.proto (hunk 1 above)
Comment (orbitalturtle) [reply proto/sensei.proto:119]: can do!

**Comment by orbitalturtle** [comment]
@johncantrell97 No problem! Added the node id to the response as well. 

Looking to add it to the http api https://github.com/L2-Technology/sensei/blob/68d53e48e0b5e7f40a711b81b15c4375fc323ccd/src/http/admin.rs#L544 But here isn't it just calling the AdminService and grabbing that response, which is already changed in the PR? Or am I missing something?

**Comment by johncantrell97** [comment]
> @johncantrell97 No problem! Added the node id to the response as well.
> 
> Looking to add it to the http api
> 
> https://github.com/L2-Technology/sensei/blob/68d53e48e0b5e7f40a711b81b15c4375fc323ccd/src/http/admin.rs#L544
> 
> But here isn't it just calling the AdminService and grabbing that response, which is already changed in the PR? Or am I missing something?

oh whoops, you're right.  looks good to me!

**Pull request merged**